i18n/en.json

```diff
@@ -3195,7 +3195,6 @@
   "mobile.set_status.offline.icon": "Offline Icon",
   "mobile.set_status.online.icon": "Online Icon",
   "modal.manual_status.ask": "Do not ask me again",
-  "modal.manual_status.auto_responder.message_": "Would you like to switch your status to \"{status}\" and disable Automatic Replies?",
```

Comment:
not sure why this change happened?

Reply:
Thanks for the feedback! When it was testing for i18n check, it was giving me an error on this line. I also did the i18n-extract localization test and it was deleting this line for some reason. I wasn't sure what else to change in my code to prevent that from deleting.

Reply:
@reflog So I changed all the 'any' types to see if the i18n check would stop deleting this line. Now I'm stumped on what to do here. Any advice?

Reply:
@kcc343 , what error were you getting when doing the i18n check? The string is currently used in webapp - can you include here the output of the "make i18n-extract" check?

Reply:
@catalintomai Here's the output I'm getting from the make i18n-extract
![18](https://user-images.githubusercontent.com/31262826/104289840-77eba600-546e-11eb-9a02-9513dcdfaa0b.JPG)

Is it not supposed to be deleting this line or it does need to be deleted?

Reply:
I don't know why "make i18n-extract" decides to remove that line. The safe solution is to keep that line and ignore the "make i18n-extract" resulting updates.